scripts/build/index.js

```diff
@@ -92,12 +103,31 @@ const copyBuildInfoToStagingDoc = () => {
   });
 };
 
+const updateServiceWorker = () => {
+  const updateSWUrl = getApiUrl('/api/v1/upgrade/service-worker');
+
+  return rpn.get(updateSWUrl).catch(err => {
+    if (err.status === 401) {
+      throw new Error('Environment variable COUCH_URL has invalid authentication');
+    }
+    if (err.status === 403) {
+      throw new Error('Environment variable COUCH_URL must have admin authentication');
+    }
+
+    if (err.error && err.error.code === 'ECONNREFUSED') {
+      console.warn('API could not be reached.');
```

Comment:
Nice to swallow this error, but I think we could have a more helpful message, eg...

```suggestion
      console.warn('API could not be reached, so the service-worker has not been updated.');
```

Reply:
This will show up in dev every time the build runs and api isn't started yet, so I had to swallow it :)